app/api/v1/models/party_models.py

```diff
@@ -140,3 +147,14 @@ def delete_party(pid):
             {"message": f"Party No. {pid} deleted succesfully"}
             ]
         }
+
+    def party_reg_validadion(self):
```

Comment:
@artorious , I think the function name should be party_reg_validation.

Reply:
Hi, I've made this change